pkg/agent/multicast/mcast_controller.go

```diff
@@ -473,6 +483,48 @@ func (c *Controller) addOrUpdateGroupEvent(e *mcastGroupEvent) {
 	}
 }
 
+func (c *Controller) memberChanged(e interface{}) {
+	podEvent := e.(types.PodUpdate)
+	namespace, name := podEvent.PodNamespace, podEvent.PodName
+
+	klog.V(2).InfoS("Pod is updated", "IsAdd", podEvent.IsAdd, "namespace", namespace,
+		"name", name)
+	g := &mcastGroupEvent{
+		group: types.McastAllHosts,
+		eType: groupJoin,
```

Comment:
No need to set `eType` since it is actually ignored by the consumer?

Reply:
done